referenceframe/model_urdf.go

```diff
@@ -9,31 +8,27 @@ import (
 	"github.com/golang/geo/r3"
 	"github.com/pkg/errors"
 
-	"go.viam.com/rdk/referenceframe"
 	"go.viam.com/rdk/spatialmath"
 	"go.viam.com/rdk/utils"
 )
 
-// Extension is the file extension associated with URDF files.
-const Extension string = "urdf"
-
-// ModelConfig represents all supported fields in a Universal Robot Description Format (URDF) file.
-type ModelConfig struct {
-	XMLName xml.Name `xml:"robot"`
-	Name    string   `xml:"name,attr"`
-	Links   []link   `xml:"link"`
-	Joints  []joint  `xml:"joint"`
+// ModelConfigXML represents all supported fields in a Universal Robot Description Format (URDF) file.
+type ModelConfigXML struct {
```

Comment:
Would `ModelConfigURDF` be a better name than `XML` given this specifically represents a URDF file (which happens to be one specific type of XML file)?

Reply:
I went back and forth on this name.  I'm down to make that change